### DISCUSSION TITLE
Progetto d'esame di Giulio Felicetti

### DISCUSSION THREAD

**Discussion by giufeli** [deneb-alpha/Lab_open_source-2024_2025#24] [Projects]
Il lavoro sarà incentrato sul progetto open source di LibreOffice, introducendo la community e testando le segnalazioni di alcuni utenti tramite Bugzilla, con l'obiettivo di contribuire alla Quality Assurance (QA) del progetto.

**Cosa tratteremo?**

- Cos'è un progetto open source.
- Cos'è una community.
- Cos'è Bugzilla.

**Cosa faremo?**

- Verificheremo i bug segnalati da altri utenti, cambiando lo stato del bug in "CONFIRMED" se riproducibile.
- Eseguiremo test sui casi segnalati, utilizzando il software per verificare i problemi riportati.

**Comment by deneb-alpha** [comment]
ciao @giufeli! 
la proposta ve bene. 

Per l'esame ricordati di prepararti anche qualche esempio di bug su cui hai già fatto il triage e che andrai a presentare "live" durante l'esame.

Immagina di fare quella presentazione come se di fronte a te ci fosse qualcuno che non ha mai sentito parlare di bug triage. Descrivi i passaggi che hai seguito per effettuare il triage e anche gli eventuali tentativi fatti.

Tra gli esempi, esponi casi dove sei stato in grado  sia di riprodurre sia di non riprodurre il bug.
